search/includes/classes/class-settingshealthjob.php

```diff
@@ -23,7 +23,17 @@ class SettingsHealthJob {
 	/**
 	 * The name of the option lock when an new index version is already being built
 	 */
-	const BUILD_LOCK_NAME = 'vip_search_new_version_building_lock';
+	const BUILD_LOCK_NAME = 'vip_search_cron_new_version_building_lock';
+
+	/**
+	 * The name of the option to store the last ID processed by the re-building job.
+	 */
+	const LAST_PROCESSED_ID = 'vip_search_cron_last_processed_id';
```

Comment:
BUILD_LOCK_NAME is using the "name" to describe itself while `LAST_PROCESSED_ID` and `BUILD_IN_PROGRESS` do not follow this same convention.